### DISCUSSION TITLE
how to add a font

### DISCUSSION THREAD

**Discussion by MACscr** [SVG-Edit/svgedit#798] [New feature request]
how do we add additional fonts? For example, I would love to have the google fonts collection available for use.

**Comment by ghost** [comment]
you can add it here:
https://github.com/SVG-Edit/svgedit/blob/master/editor/svg-editor.html#L359

**Reply by ericmp33** [reply]
Link leads to 404

**Comment by brettz9** [comment]
Having a feature to retain fonts in local storage sounds like a useful feature... Beyond NeiroNx's suggestion, would that meet your needs?

Btw, does anyone know anything about browser support with `@font-face` and SVG `text`?

**Comment by cuixiping** [comment]
Font files can be saved into `IndexedDB`. Adding font files also needs to consider pdf export.

**Comment by brettz9** [comment]
Yeah, IndexedDB does sound good--I was speaking more generically about "local storage" rather than `localStorage`. :) But all something someone else will probably have to implement :)

**Comment by jfhenon** [comment]
Adding fonts may be a useful feature. We would welcome a PR about this.

**Comment by ndunnaoc** [comment]
Any news on this feature?

**Comment by jfhenon** [comment]
Hello, I think we should look into this for version 7.5 (I can't give you a date).

**Comment by ericmp33** [comment]
Is there any updates on this? I'm trying to find a way to add a font manually.

**Comment by jfhenon** [comment]
The current way to do this is to modify lines 149,150 in TopPanel.html to add your own fonts:
````
 <!-- Font Size-->
    <se-select id="tool_font_family" label="properties.font_family_label"
options="properties.serif,properties.sans_serif,properties.cursive,properties.fantasy,properties.monospace,properties.courier,properties.helvetica,properties.times,properties.futura, properties.wingdings"
      values="Serif::Sans-serif::Cursive::Fantasy::Monospace::Courier::Helvetica::Times::Futura::Wingdings"></se-select>
 ````
You also need to add the translation of the font name in the locale folder (at least for english).

I just tested it and this seems to work correctly.

**Comment by lazytyper** [comment]
Today I discovered that it is now also possible to access local fonts.

See https://wicg.github.io/local-font-access/

**Reply by cuixiping** [reply]
Good news!

> Today I discovered that it is now also possible to access local fonts.
> 
> See https://wicg.github.io/local-font-access/
